assets/app/view/game/round/auction.rb

```diff
@@ -56,12 +56,16 @@ def render_players
         end
 
         def render_company_pending_par
-          corporation = @step.company_pending_par.abilities(:share).share.corporation
+          render = []
```

Comment:
call this children because render is a method name in snabberb::component

Reply:
Change applied.